new_dawn_server/users/api/resources.py

```diff
@@ -279,6 +280,16 @@ class Meta:
         queryset = Profile.objects.all()
         resource_name = "profile"
 
+    def apply_filters(self, request, applicable_filters):
+        limit = 3
```

Comment:
Let’s define a constant at the top of the file (MAX_NUM_PROFILES = 5) And use that here.